test/test_posv.cc

```diff
@@ -54,7 +54,10 @@ void test_posv_work(Params& params, bool run)
     slate::Method methodHemm = params.method_hemm();
 
     // Currently only posv* supports timer_level >= 2.
-    if (params.routine != "posv")
+    std::vector<std::string> timer_lvl_support{ "posv", "posv_mixed", "posv_mixed_gmres" };
+    bool supported = std::find(timer_lvl_support.begin(), timer_lvl_support.end(), params.routine) != timer_lvl_support.end();
+
+    if (supported == false)
```

Comment:
This is where I really like spaces inside parens, to avoid 35 chars without space. Also wrap to 80 chars. The [style guide discourages comparison with `true` and `false`](https://github.com/icl-utk-edu/slate/wiki/code-style-guide#pet-peeves). Maybe like:
```
    bool supported = std::find( timer_lvl_support.begin(),
                                timer_lvl_support.end(), params.routine )
                     != timer_lvl_support.end();
    if (! supported)
```